js/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-- Node.JS 4,5, or 6+
```

Comment:
Correct but Node.js 4 will still be supported and people might want to run the benchmarks for it. On top of this the fix to add compatibility for Node.js 4 is trivial.